tests/test_wave_03.py

```diff
@@ -54,12 +54,14 @@ def test_friends_unique_movies_not_duplicated():
 
     # Arrange
     assert len(friends_unique_movies) == 3
-
+    assert INTRIGUE_3 in friends_unique_movies
```

Comment:
Nice use of `in` to make the test more flexible than relying on hard-coded indices!